packages/blocks/table/package.json

```diff
@@ -81,6 +81,95 @@
   "blockprotocol": {
     "displayName": "Table",
     "icon": "public/table.svg",
-    "image": "public/preview.svg"
+    "image": "public/preview.svg",
+    "examples": [
+      {
+        "data": {
+          "__linkedData": {
+            "aggregate": {
+              "itemsPerPage": 5,
+              "entityTypeId": "d399c523-e152-49ed-81a4-56d9e17196bb",
+              "pageNumber": 1,
+              "pageCount": 1
+            },
+            "entityTypeId": "d399c523-e152-49ed-81a4-56d9e17196bb"
+          },
+          "data": [
+            {
+              "id": "73d4be68-bf30-4af4-b15d-645dd1dedd9a",
+              "entityId": "a95813d4-6ee9-482c-8f92-b33d2f481f5c",
```

Comment:
Can we make these much shorter? It'll look better in the table. Maybe just `person1`, etc